vivisect/parsers/blob.py

```diff
@@ -10,11 +10,12 @@
     'arm':'armcall',
     }
 
-def parseFd(vw, fd, filename=None):
+def parseFd(vw, fd, filename=None, baseaddr=None):
     fd.seek(0)
     arch = vw.config.viv.parsers.blob.arch
     bigend = vw.config.viv.parsers.blob.bigend
-    baseaddr = vw.config.viv.parsers.blob.baseaddr
+    if baseaddr == None:
```

Comment:
It's more pythonic to say `if baseaddr is None:`, and since we have a mixed use case in the codebase already, we should probably standardize on being roughly flake8 compliant